FLASK/Урок 4. Введение в многозадачность/app.py

```diff
@@ -0,0 +1,107 @@
+import os
+from pathlib import Path
+import requests
+import time
+import argparse
+import threading
+from multiprocessing import Process
+import asyncio
+import aiohttp
+
+images = []
+with open('images.txt', 'r') as f:
+    for image in f.readlines():
+        images.append(image.strip())
+
+PATH = Path('images')
+
+
+def download_img(url, dir_path=PATH):
+    start_time = time.time()
+    response = requests.get(url)
+    filename = url.split('/')[-1]
```

Comment:
Лучше делить один раз
```suggestion
    filename = url.rsplit('/', 1)[-1]
```
или
```suggestion
    filename = os.path.basename(url)
```
Последний вариант предпочтительней. Так как Будет работать работать на любой платформе.